frontend/src/pages/PipelineDetails.tsx

```diff
@@ -561,8 +561,11 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
 
   public async handleVersionSelected(versionId: string): Promise<void> {
     if (this.state.v1Pipeline) {
-      const selectedVersion = (this.state.v1Versions || []).find(v => v.id === versionId);
-      const pageTitle = this.state.v1Pipeline.name?.concat(' (', selectedVersion?.name!, ')');
+      const selectedVersionV1 = (this.state.v1Versions || []).find(v => v.id === versionId);
+      const selectedVersionV2 = (this.state.v2Versions || []).find(
+        v => v.pipeline_version_id === versionId,
+      );
+      const pageTitle = this.state.v1Pipeline.name?.concat(' (', selectedVersionV1?.name!, ')');
```

Comment:
We only show version name if there's selectedVersionV1, but not show it when selected version is v2?

Reply:
Actually, `selectedVersionV1.name` is identical to `selectedVersionV2.display_name`. We choose to use `selectedVersionV1.name`, because this is helper function for `PipelineDetailsV1`. That is , this confusion(?) will be avoided after refactoring work.